vendor/hwservice.te

```diff
@@ -1,3 +1,6 @@
 type hal_display_config_hwservice, hwservice_manager_type;
 type nxpnfc_hwservice,             hwservice_manager_type;
 type nxpese_hwservice,             hwservice_manager_type;
+type vnd_qcrilhook_hwservice,      hwservice_manager_type;
```

Comment:
I would name it `vnd_qcril_audio_hwservice`  like marlin, easier to associate with our qcrilam service.

Reply:
follow crosshatch since we need maintain the line with sdm845 code